validation/tcd_latest.xsd

```diff
@@ -30,6 +30,9 @@
       <xs:element minOccurs="0" name="show-uncommitted-data-checkbox" type="BooleanProperty-CT"/>
       <xs:element minOccurs="0" name="ssl-requires-certificate" type="BooleanProperty-CT"/>
       <xs:element minOccurs="0" name="warehouse-prompt" type="TextField-CT"/>
+      <xs:element minOccurs="0" name="vendor1-prompt" type="TextField-CT"/>
```

Comment:
Could these 3 lines be placed above the  line
 `<xs:element minOccurs="0" name="warehouse-prompt" type="TextField-CT"/>` 
so that they are in alphabetical order based on `name` ? Otherwise looks good !

Reply:
Reordered